### PR TITLE
fixed vcdl-864

### PR DESCRIPTION
**Pull request by syedQuadri7**

### CODE REVIEW

**Review by thomashashi — COMMENTED** [review]
Going to put some comments in VCDL-846 - this looks fine, but also looks like it's just from a version 2 -> version 3 conversion, so I want to make sure I didn't miss the other fixes required for this ticket.

File: instruqt-tracks/terraform-intro-aws/01-setup-our-environment/assignment.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+slug: setup-our-environment
+id: ebj23ohhehzw
```
Comment (jennamwong) [inline instruqt-tracks/terraform-intro-aws/01-setup-our-environment/assignment.md:3]: Take out all IDs, then good to go!

Comment (syedQuadri7) [inline instruqt-tracks/terraform-intro-aws/01-setup-our-environment/assignment.md:3]:
```suggestion
```

File: instruqt-tracks/terraform-intro-aws/02-hello-terraform/assignment.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+slug: hello-terraform
+id: e0tq7efazoo2
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-intro-aws/02-hello-terraform/assignment.md:3]:
```suggestion
```

**Review by jennamwong — APPROVED** [review]
LGTM

**Pull request merged**